spec/features/user_creates_new_book_spec.rb

```diff
@@ -6,6 +6,7 @@
 
     create_book('The Fellowship of the Ring', 'J. R. R. Tolkien' )
 
+    expect(page.current_path).to eq books_path
```

Comment:
Instead of inspecting the path, it is usually best to test what the user will see on the page. So in this case you may want test that the books index header is here, or you can test that there is css like `.books-rows` or something